code-challenges.js

```diff
@@ -9,67 +9,156 @@
-var temp3 = 212
-
+// var temp1 = 85
+// var temp2 = 350
+// var temp3 = 212
 
+// const temperatureOutcome = (number) => {
+//       if (number < 212) {
+//         return `${number} is below boiling point`
+//     } else if (number === 212) {
+//         return `${number} is at boiling point`
+//     } else if (number > 212) {
+//         return `${number} is above boiling point`
+//     }
+// }
 
+// console.log(temperatureOutcome(temp3))
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45] [10, 25, -40, 100, 160]
 
-var myForNumbers1 = [3, 7, 0, 6, -9]
-var myForNumbers2 = [2, 5, -8, 20, 32]
-
-
-
+// var myForNumbers1 = [3, 7, 0, 6, -9]
+// var myForNumbers2 = [2, 5, -8, 20, 32]
+// create a function that multiplies each number in arrays by 5
+// use a for loop
+// i need it to return the arrays multiplied by 5!!!!!!!!!!!
+// const numbersMultiplied = (array) => {
+//     let numbers = []
+//     for (let i = 0; i < array.length; i++) {
+//         numbers.push(array[i] * 5)
+// }
+// return numbers
+// }
+// console.log(numbersMultiplied(myForNumbers1))
+// console.log(numbersMultiplied(myForNumbers2))
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
 
-var myMapNumbers1 = [87, 6, 28, 16, 2]
-var myMapNumbers2 = [8, -7, 0, 6, 12]
+// write the variables
+// var myMapNumbers1 = [87, 6, 28, 16, 2]
+// var myMapNumbers2 = [8, -7, 0, 6, 12]
+// // write the function that will pull each number and multiply it by 5 using map
+// // const mappedNums = myMapNumbers1.map(numbers =>  {
+// //       return numbers * 5
+// //         // get the return and multiply it by 5
+// // })
 
+// // console.log(mappedNums)
+// // I DID IT WITH ONE ARRAY BUT CANT FIGURE OUT HOW TO GET BOTH, LET ME TRY AGAIN
 
+// const  mappedNums = (array) => {
+//     return array.map(value => {
+//         return value * 5
+//     })
+// }
 
+// console.log(mappedNums(myMapNumbers2))
 
+// I THINK I GOT IT!!!! I WAS HAVING TROUBLE FIGURING OUT WHERE TO PUT MY (ARRAY) AND WHERE TO PUT MY (VALUE)
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
-
+const removedVowels = (array) => {
+// // 	// Create variable to store the letters
+    let letters = array.split(""); 
+    let letters2 =letters.filter(value => {
+        if (! ["a", "e", "i", "o", "u"].includes(value)) {
+            return true
+        } else {
+            return false
+        }
+    })
+    return letters2.join('')
+}
+console.log(removedVowels(removedVowels(stringWithVowels2)))
```

Comment:
you should only need to call this function once 
```suggestion
console.log(removedVowels(stringWithVowels2))
```